### PR TITLE
Add Kongress flatpak manifest

### PR DESCRIPTION
**Pull request by dimkard**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions.
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
_I am the author and maintainer of the application._
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*
_No additional patches needed yet_
[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by barthalion** [comment]
bot, build org.kde.kongress

**Comment by barthalion** [comment]
cc @flathub/kde

File: org.kde.kongress.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--device=dri",
+        "--share=network",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.Solid.PowerManagement",
+        "--own-name=org.kde.kongressac"
+    ],
+    "modules": [
```
Comment (barthalion) [inline org.kde.kongress.json:17]: You should add a global or per-module `cleanup` for deleting static libraries and development headers.

**Comment by nedrichards** [comment]
@flathub/kde FYI

**Comment by cho2** [comment]
Too many messages, maybe from Kirigami side?
```
flatpak run org.kde.kongress
Qt: Session management error: None of the authentication protocols specified are supported
Calendar  "fosdem2021"  loaded succesfully
Calendar  "las2020"  loaded succesfully
Calendar  "fosdem2021"  loaded succesfully
Calendar  "las2020"  loaded succesfully
file:///usr/lib/qml/org/kde/kirigami.2/GlobalDrawer.qml:265: TypeError: Type error
No sufficient calendar information provided
No sufficient calendar information provided
file:///usr/lib/qml/org/kde/kirigami.2/private/RefreshableScrollView.qml:137:13: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
file:///usr/lib/qml/org/kde/kirigami.2/templates/AbstractApplicationHeader.qml:162:9: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
file:///usr/lib/qml/org/kde/kirigami.2/templates/AbstractApplicationHeader.qml:99:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
file:///usr/lib/qml/org/kde/kirigami.2/templates/AbstractApplicationHeader.qml:139:9: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
file:///usr/lib/qml/org/kde/kirigami.2/templates/AbstractApplicationHeader.qml:162:9: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
file:///usr/lib/qml/org/kde/kirigami.2/templates/AbstractApplicationHeader.qml:139:9: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
file:///usr/lib/qml/org/kde/kirigami.2/templates/AbstractApplicationHeader.qml:99:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
file:///usr/lib/qml/org/kde/kirigami.2/templates/OverlayDrawer.qml:284:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
file:///usr/lib/qml/org/kde/kirigami.2/templates/AbstractApplicationHeader.qml:162:9: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
file:///usr/lib/qml/org/kde/kirigami.2/templates/AbstractApplicationHeader.qml:139:9: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
file:///usr/lib/qml/org/kde/kirigami.2/templates/AbstractApplicationHeader.qml:99:5: QML Connections: Implicitly defined onFoo properties in Connections are deprecated. Use this syntax instead: function onFoo(<arguments>) { ... }
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
file:///usr/lib/qml/org/kde/kirigami.2/private/globaltoolbar/PageRowGlobalToolBarUI.qml:65:9: QML RowLayout (parent or ancestor of QQuickLayoutAttached): Binding loop detected for property "maximumWidth"
```

File: org.kde.kongress.json (hunk 1 above)
Comment (dimkard) [reply org.kde.kongress.json:17]: Static libraries and headers cleanup 96fe84c

File: org.kde.kongress.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        {
+            "name": "kcalendarcore",
```
Comment (tsdgeos) [inline org.kde.kongress.json:19]: you should not need to build calendarcore, it's part of the sdk

**Comment by nedrichards** [comment]
bot, build org.kde.kongress

File: org.kde.kongress.json (hunk 2 above)
Comment (dimkard) [reply org.kde.kongress.json]: Done

**Comment by barthalion** [comment]
bot, build org.kde.kongress

**Comment by tsdgeos** [comment]
Looks good to me

**Pull request closed without merging**